internal/provisioner/kops_provisioner_cluster_installation.go

```diff
@@ -627,48 +627,73 @@ func (provisioner *crProvisionerWrapper) getMattermostCustomResource(cluster *mo
 	return cr, nil
 }
 
-// ExecMattermostCLI invokes the Mattermost CLI for the given cluster installation with the given args.
+// ExecMattermostCLI invokes the Mattermost CLI for the given cluster installation
+// with the given args. Setup and exec errors both result in a single return error.
 func (provisioner *KopsProvisioner) ExecMattermostCLI(cluster *model.Cluster, clusterInstallation *model.ClusterInstallation, args ...string) ([]byte, error) {
-	return provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, append([]string{"./bin/mattermost"}, args...)...)
+	output, execErr, err := provisioner.ExecClusterInstallationCLI(cluster, clusterInstallation, append([]string{"./bin/mattermost"}, args...)...)
+	if err != nil {
+		return output, errors.Wrap(err, "failed to setup mattermost exec command")
+	}
+	if execErr != nil {
+		return output, errors.Wrap(execErr, "failed to run mattermost exec command")
```

Comment:
Perhaps we could adjust the error a bit, for the first one: `failed to run mattermost command in the container...` and for the second `mattermost command failed with error...`? 0/5